core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3303,6 +3304,14 @@ public String toString() {
           .setIsHidden(true)
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .build();
+  public static final PropertyKey MASTER_METADATA_SYNC_PENDING_PATH_TYPE =
```

Comment:
I propose `MASTER_METADATA_TRAVERSE_TYPE = DFS/BFS` where we can say `DFS consumes less memory while BFS is more fair for all concurrent sync tasks`